client/dashboard/sites/settings-wpcom-login/index.tsx

```diff
@@ -0,0 +1,47 @@
+import { isEnabled } from '@automattic/calypso-config';
+import { useSuspenseQuery } from '@tanstack/react-query';
+import { notFound } from '@tanstack/react-router';
+import { createInterpolateElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { siteBySlugQuery } from '../../app/queries/site';
+import InlineSupportLink from '../../components/inline-support-link';
+import PageLayout from '../../components/page-layout';
+import { HostingFeatures } from '../../data/constants';
+import HostingFeatureGatedWithCallout from '../hosting-feature-gated-with-callout';
+import SettingsPageHeader from '../settings-page-header';
+import SsoForm from './sso-form';
+
+export default function WpcomLoginSettings( { siteSlug }: { siteSlug: string } ) {
+	const { data: site } = useSuspenseQuery( siteBySlugQuery( siteSlug ) );
+
+	if ( ! isEnabled( 'dashboard/v2/security-settings' ) ) {
+		throw notFound();
+	}
+
+	return (
+		<PageLayout
+			size="small"
+			header={
+				<SettingsPageHeader
+					title={ __( 'WordPress.com log in' ) }
```

Comment:
I think it should be `login` as a noun 🤔 but I see that our support doc uses `log in`, so I'm torn. 😄

Reply:
FWIW, the naming in not consistent either between WP.com (the old Settings > Security page) and Jetpack:

WP.com | Jetpack
--- | ---
<img width="744" height="258" alt="Screenshot 2025-08-13 at 10 03 23" src="https://github.com/user-attachments/assets/3892ff9a-d688-4672-b704-e3ee646d4d38" /> | <img width="1073" height="279" alt="Screenshot 2025-08-13 at 10 02 56" src="https://github.com/user-attachments/assets/cc88ce67-f3f3-4eb6-88fb-9bb1e006ce28" />

I think it should be `login` too, but it's not something that we're introducing in this PR, so maybe can be checked/addressed later.

Reply:
Hosting Dashboard feels like an opportunity to be consistent in our language :)